test/features/projects/deleting_a_project_test.rb

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+feature "Deleting a project" do
+  scenario "is successful" do
+    project = projects(:one)
+    visit project_path(project)
+    click_link 'Destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.